addons/invtracker/README.md

```diff
@@ -0,0 +1,17 @@
+# invtracker
+This addon displays a grid detailing empty and filled inventory slots, similar to the FFXIV Inventory Grid HUD widget.
+
+![Imgur](https://i.imgur.com/PgiMxRZ.png)
+
+## How to edit the settings
+1. Login to your character in FFXI
+2. Edit the addon settings file: **_Windower4\addons\invtracker\data\settings.xml_**
+3. Save the file
+4. Press Insert in FFXI to access the windower console
+5. Type ``` lua r invtracker ``` to reload the addon
+6. Press Insert in FFXI again to close the windower console
+
+## Issues:
+1. You may need to log out from you character and enter again in order to have the addon display properly if loaded manually.
```

Comment:
For some reason, the images are not loaded properly when loading the addon from within the game, if you look closely at the picture they look like small dots instead of squares: https://imgur.com/iT5iQqP

Zoning solves the issue, so i've updated the readme with your suggestion.

Reply:
This is something that should be looked into, that really shouldn't happen. If you can narrow down the issue or create a small test case that doesn't work right, I can look into it.